### ISSUE TITLE
Parameter for GPU memory

### ISSUE THREAD

**Issue by rohittidke** [Azure/azure-cli#6831]
please provide support for GPU as a parameter in creating a container instance. Most of the containers we spin need GPU support. Thanks! 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2e5d1da7-7c4f-c9e6-e4b9-a30407675acf
* Version Independent ID: ba32e12d-8283-f980-e1d0-1ca2a75a5152
* Content: [az container](https://docs.microsoft.com/en-us/cli/azure/container?view=azure-cli-latest#az_container_create)
* Content Source: [src/command_modules/azure-cli-container/azure/cli/command_modules/container/_help.py](https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-container/azure/cli/command_modules/container/_help.py)
* Service: **container-service**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by jluk** [comment]
@rohittidke we're evaluating this support right now, could you share your workload scenario and expected scale? Any details are helpful for us. 

Please head over to aka.ms/aci/feedback as well for other feature requests.

**Comment by rohittidke** [comment]
Hallo Justin,

I am working with Avanade. Usually we have a web application connecting to
multiple container instances for testing and demo scerarios. We tried out
the Azure Web App on a container for our containers but there were several
issues so we tried out container instances later and it worked. The problem
we faced is the deploying a cluster for the same scenario turned out to
bevery expensive. The problem is for simple NLP tasks and classification
algorithms even a SVM run very slowly on the container instance. A single
web service call for the simplest classification takes more than 1000ms.

So in summary, we need an affordable platform for our micro-service
architecture consisting of machine learning components.

Please inform us if we should use any other Azure Product and have
misunderstood the use of Container Instances.

Best regards,

Rohit Tidke



Am Di., 7. Aug. 2018 um 19:10 Uhr schrieb Justin Luk <
notifications@github.com>:

**Comment by jluk** [comment]
Thanks @rohittidke this is really helpful! I think ACI can help port over simple NLP tasks to be run, are any of these time sensitive? Would a start-time in the range of 10-15 minutes be acceptable in this situation?

For the output of your task is that being written out to some external DB or a volume mount?

**Comment by rayoef** [comment]
#reassign:tomvcassidy

**Comment by rayoef** [comment]
#label:"container-instances"

**Comment by wangzelin007** [comment]
#reassign:tomvcassidy

**Comment by wangzelin007** [comment]
#label:"container-instances"

**Comment by jsntcy** [comment]
#reassign:tomvcassidy

**Comment by jsntcy** [comment]
#reassign:jsntcy

**Comment by wangzelin007** [comment]
Hi @rayoef,
Do you want to replace label `Container Instances` with `container-instances` permanently?
It seems you are trying to make changes to the issue by commenting with `#` tag
I'm very curious in which repo you've used similar feature before, it doesn't seem to be native to github.

**Comment by rayoef** [comment]
Hi @wangzelin007, I want to remove the container-service label from the original document details. Do you know how to do that? The #label tag is used in azure-docs-pr.

**Comment by wangzelin007** [comment]
Hi @rayoef,
Sorry, I don't quite understand what the original document details refer to. 
If it is to remove the label in this issue, I can handle it manually.

**Comment by wangzelin007** [comment]
1. The `#label` tag in azure-docs-pr is to add labels, not to remove label.
2. This function is developed by the docs team, but cli does not support it.
3. container-service does not appear in this issue, where do you want to remove the container-service label ?
![image](https://github.com/Azure/azure-cli/assets/18628534/ceafe012-8143-4b67-b817-ec5287b546be)

**Comment by wangzelin007** [comment]
Hi @rayoef,
I'm still confused, I could edit the issue and delete the line `Service: container-service`.
But I don't think this is what you want, simply deleting this text doesn't change anything.
Do you expect to remove the `Service: container-service` related content in azure-docs-pr ?
Do you know how `Document Details` is generated ?
May I ask why you need to remove the `Service: container-service` ?
